RecoEgamma/EgammaTools/python/slimmedEgammaFromMultiCl_cff.py

```diff
@@ -42,7 +44,7 @@
     src = cms.InputTag("patElectronsFromMultiCl"),
     cut = cms.string("!isEB && pt >= 10."),
 )
-slimmedElectronsFromMultiCl = cms.EDProducer("PATElectronSlimmer",
+slimmedElectronsFromMultiCl = slimmedElectrons.clone(
     src = cms.InputTag("selectedPatElectronsFromMultiCl"),
     dropSuperCluster = cms.string("0"),
     dropBasicClusters = cms.string("0"),
```

Comment:
please eliminate common parameters from the list (this was a part of the original comment on this subject)